i18next-parser.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  locales: ["en", "ja"],
+  output: "src/translations/$LOCALE.yml",
```

Comment:
Is there any reason we will keep translations separated from i18n code? In CMS and Cloud we are storing everything together, so if there is a good reason to keep them separate I will update CMS and cloud to keep things streamlined. If not, please output to 
```suggestion
  output: "src/i18n/translations/$LOCALE.yml",
```